back/src/Guide/Cli.hs

```diff
@@ -24,42 +24,10 @@ import qualified Options.Applicative as Opt
 
 -- | All available commands
 data Command
-    = RunServer             -- ^ run server
-    | DryRun                -- ^ load database and exit
-    | LoadPublic FilePath   -- ^ load PublicDB, create base on it and exit
-
-----------------------------------------------------------------------------
--- Parsers
-----------------------------------------------------------------------------
-
-{-
```

Comment:
I have removed this section because we shouldn't have to update it every time the syntax is changed.